src/main/java/io/swagger/oas/inflector/controllers/OpenAPIOperationController.java

```diff
@@ -216,7 +216,7 @@ public Response apply(ContainerRequestContext ctx) {
         List<Parameter> parameters = operation.getParameters();
         int requestBody = 0;
         if(operation.getRequestBody() != null){
```

Comment:
request body from operation it's not used in if-block, why the null check is required for it?